src/components/Anchor/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+/**
+ * This constant is used to correct the scroll bar position when navigating to an anchor
+ * Since we want the target element to appear below the scrollbar
+ */
+const TOP_BAR_HEIGHT = 110;
```

Comment:
Can we get this with JS? That way, if we change the height of the navbar in the future, we don't have to manually update it here.